src/components/Navigation.css

```diff
@@ -0,0 +1,9 @@
+.Nav {
+  display: flex;
+  justify-content: space-around;
+}
+
+.Navlink {
+  text-decoration: none;
+  color: #ffffff;
+}
```

Comment:
maybe you use a predefined color palette, so you dont hardcode everytime the code in css.

Reply:
I should most definitely do that too.